uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Reminder.php

```diff
@@ -1067,4 +1067,18 @@ private static function _getTaskIncompleteStatuses() {
     return $taskIncompleteStatuses;
   }
 
+  /**
+   * Checks if an Activity Type belongs to a given component,
+   * for example, "CiviTask" or "CiviDocument"
+   *
+   * @param int $activityTypeId
+   * @param string $componentName
+   * @return boolean
+   */
+  private static function _checkIfActivityTypeBelongsToComponent($activityTypeId, $componentName) {
```

Comment:
No need to prefix function name with underscore.

Reply:
Done.